### PR TITLE
Completed css-layout-classes exercise. - Jeff Jin

### PR DESCRIPTION
**Pull request by jeffmjin**
Screen Shot 2022-08-25 at 1 58 17 PM

Updated screenshot

Screen Shot 2022-08-25 at 2 55 05 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            </p>
+          </div>
+          <div class="column-one-third">
+            <img src="./images/yoshi.png">
+          </div>
+        </div>
+      </article>
+    </div>
+    <div class="column-full">
+      <article>
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: I would remove the column full and article elements here, you just want the `row` class with the four column one four elements

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <header>
+          <h1>Choose your character!</h1>
+        </header>
+      </div>
+    </div>
+    <div class="column-full">
+      <article>
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: I would remove the column full and article elements here.

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+          </div>
+          <div class="column-one-fourth">
+            <img src="./images/ness.png">
+          </div>
+        </div>
+      </article>
+    </div>
+    <div class="row">
+      <div class="column-two-thirds">
+        <article>
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: this article element is not needed, although its not hurting anything!

File: css-layout-classes/index.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+      <div class="column-one-third">
+        <img src="./images/dk.png">
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <img src="./images/mario.png">
+      </div>
+      <div class="column-two-thirds">
+        <article>
```
Comment (BrettAlbright) [inline css-layout-classes/index.html]: same as above, this element is not needed, but it is not hurting anything either!

File: css-layout-classes/styles.css [hunk 5]
```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+img {
+  width: 100%;
+}
+
+.container {
+  width: 90%;
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css]: as per the hints in the instructions, this should be set to a max-width of 800px and you should have margin set to 0 auto to center the container on the page

File: css-layout-classes/styles.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+.column-one-third {
+  width: 20%;
+}
+
+.column-two-thirds {
+  width: 35%;
+}
+
+.column-one-fourth {
```
Comment (BrettAlbright) [inline css-layout-classes/styles.css:28]: the width set for the columns should match the semantic name, so one fourth should be 25%, one third should be `calc(100% / 3)` two thirds should be `calc(200% / 3)` so on and so fourth

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
coming together! Just a few updates to get. Make sure to reach out if you have questions on my comments!

**Review by Zhiwutian — APPROVED** [review]
Great work Jeff!

**Pull request merged**